agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -552,6 +556,18 @@ class backhaul_manager : public btl::transport_socket_thread {
     std::vector<sApMetricsQuery> m_ap_metric_query;
     std::vector<sApMetricsResponse> m_ap_metric_response;
 
+    struct sChannelSelectionResponse {
+        sMacAddr radio_mac;
+        uint8_t response_code;
```

Comment:
```suggestion
         wfa_map::tlvChannelSelectionResponse::eResponseCode response_code;
```

With this change you save two type-casts in .cpp file

Reply:
fixed